tests/config.test.ini

```diff
@@ -108,6 +108,8 @@ elasticsearch=http://127.0.0.1:8200
 multiES=true
 multiESPort=8200
 multiESNodes=127.0.0.1:9200,prefix:tests;127.0.0.1:9200,prefix:tests2_
+multiESClusters=test;test2
+isLocalViewRegExp=testmulties
```

Comment:
why is this needed?

Reply:
`isLocalViewRegExp` is needed to test multiviewer locally when running `npm run viewer:multiviewer` and `npm run viewer:testmultiviewer`.

Reply:
Hmm ok, something else must be wrong because it shouldn't, I'll take a look.